matcher/dashboard/views/imports.py

```diff
@@ -106,3 +107,20 @@ def dispatch_request(self, id):
         ctx["file"] = file
         ctx["form"] = form
         return render_template("imports/show.html", **ctx)
+
+
+class DownloadImportFileView(View, DbMixin):
+    def dispatch_request(self, id):
+        import_file = self.query(ImportFile).get_or_404(id)
+
+        # FIXME: not every client supports gzip, we should look at the Accept-Encoding header
+        response = send_file(
+            import_file.open(),
+            mimetype="text/csv",
+            as_attachment=True,
+            attachment_filename=os.path.split(import_file.path)[-1],
+        )
+        response.headers["Content-Encoding"] = "gzip"
```

Comment:
Sending as gzip but the content is actually never gzipped (unlike export files).
Quick fix: remove this line. Better fix: gzip the file before sending it, but that’s a bit tricky (and even export files are not gzipped on the fly) so that is probably something to defer for now.

Reply:
done

Reply:
:+1: